From: Mehdi Dogguy <mehdi@debian.org>
Date: Sun, 25 Apr 2010 14:50:30 +0200
Subject: prefix binary names with 'ocaml' and do not install useless files

---
 Makefile.in | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index fb2cbc3..0d19aaf 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -389,11 +389,13 @@ install: install-$(OCAMLBEST) install-byte
 	mkdir -p $(BINDIR)
 ifeq (@LABLGNOMECANVAS@,yes)
 ifeq ($(OCAMLBEST),byte)
-	cp -f $(BINDIR)/graph-editor.byte $(BINDIR)/graph-editor$(EXE)
-	cp -f $(BINDIR)/graph-viewer.byte $(BINDIR)/graph-viewer$(EXE)
+	mv $(BINDIR)/graph-editor.byte $(BINDIR)/ocamlgraph-editor$(EXE)
+	mv $(BINDIR)/graph-viewer.byte $(BINDIR)/ocamlgraph-viewer$(EXE)
 else
-	cp -f $(BINDIR)/graph-editor.opt $(BINDIR)/graph-editor$(EXE)
-	cp -f $(BINDIR)/graph-viewer.opt $(BINDIR)/graph-viewer$(EXE)
+	mv $(BINDIR)/graph-editor.opt $(BINDIR)/ocamlgraph-editor$(EXE)
+	mv $(BINDIR)/graph-viewer.opt $(BINDIR)/ocamlgraph-viewer$(EXE)
+	mv $(BINDIR)/graph-editor.byte $(BINDIR)/ocamlgraph-editor.byte
+	mv $(BINDIR)/graph-viewer.byte $(BINDIR)/ocamlgraph-viewer.byte
 endif
 endif
 
